### ISSUE TITLE
Simplify `bref cli`: avoid having to pass the function name

### ISSUE THREAD

**Issue by mnapoli** [brefphp/bref#125]
Follow up of #122

The `bref cli` helper requires to pass the name of the function to invoke:

```
vendor/bin/bref cli <function name> -- doctrine:migrate
```

Ideally we should run this:

```
vendor/bin/bref cli -- doctrine:migrate
```

It should:

- autodetect the function to call based on `serverless.yml`
- autodetect the region to use

**Comment by Guillaume-Rossignol** [comment]
The region detection is done in #949. I don't like the idea to autodetect the function to call (because sometimes I don't use «framework cli» but some php file write by hand). Maybe we can use the function `cli` by default if it's defined in `serverless.yml` ?

**Comment by t-richard** [comment]
Like the idea to simplify it. 

Having a default function name like `cli` would imply to change it throughout the docs and packages to make it the default in the provided serverless.yml (laravel bridge, Symfony recipe, console docs and maybe elsewhere) to provide the best DX.

I personally would go with `console` as a default name because it's also the name of the layer.

**Comment by Guillaume-Rossignol** [comment]
It doesn't seems to be hard to implement (https://github.com/Guillaume-Rossignol/bref/commit/3febbcab1faed9e14b072e384bcce2ecc76353e4) but I don't feel confortable with the laravel/Symfony documentation.
On second thought, it doesn't break backward compatibility. So I could make the PR and see later for the doc :roll_eyes:

**Comment by t-richard** [comment]
Using console as a default name would imply no changes to symfony docs neither symfony recipe.

For laravel that would mean to change [this line](https://github.com/brefphp/laravel-bridge/blob/master/config/serverless.yml#L30) to `console` instead of `artisan`.

Not sure if that is impacting something else (and again by impacting I mean that it would be great to benefit from this from a DX perspective).

**Comment by mnapoli** [comment]
Fixed in v2 via #1303
